distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Range.enso

```diff
@@ -214,7 +217,7 @@ type Date_Range
              (Date.new 2021 05 07).up_to (Date.new 2021 05 10) . to_vector
     to_vector : Vector Date
     to_vector self =
-        proxy = Array_Proxy.new self.length self.at
+        proxy = Array_Proxy.new self.length (self.at _)
```

Comment:
What does this change do? Naively, I thought `self.at` == `x-> self.at x` == `self.at _`.

Reply:
Once the index argument of `at` got a default, calling `self.at` will force all default arguments and thus will return the first element, not the lambda. The equation above holds if there are no default arguments.

Btw.
```suggestion
        proxy = Array_Proxy.new self.length (self.at ...)
```
would probably work as well